app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/HorizontalSnapScrollView.java

```diff
@@ -198,6 +198,9 @@ public static int calcMaxCols(Resources res, int availPixels) {
 
     // FIXME: Landscape patch to expand a column to full width if there is space available
     private static int calcMaxCols(Resources res, int availPixels, int columnsCount) {
+        if (columnsCount < 0) {
+            columnsCount = 0;
+        }
```

Comment:
This feels like the wrong fix. My suggestion would be to either not call `calcMaxCols()` when `HorizontalSnapScrollView.roomsCount` is not initialized or pass 0 to this method. Doing it here is confusing.

This being a problem also triggers the question what happens when `setRoomsCount()` is called after `onSizeChanged()` has already been called. Currently I don't see how this would trigger the column width to be recalculated.

Reply:
Thanks. Changed in c57867f3b3028cdc53f21d7694428253a33365df.

Reply:
I suggest to continue discussing your second point outside of this bugfix since I want to release it asap. There is certainly room for improvement. Right now `getColumnWidth()` is making publicly available what is calculated in `onSizeChanged()`. Therefore, `getColumnWidth()` contains a check to make sure `setRoomsCount()` has been invoked before.

Reply:
I created issue #275 which might be related to your comment - I am not sure right now.